### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by vlados** [fmtl-studio/laravel-borica#2]
Can you please add more documentation on using the package with simple code

**Comment by konstantinrachev** [comment]
Yep, will do this as soon as days or two from now. Laravel 8 will be supported as well. Thanks.

**Comment by vlados** [comment]
@konstantinrachev Any updates? Any simple code will do the trick
